### PR TITLE
gammu: 1.33.0 -> 1.38.2

### PR DESCRIPTION
**Pull request by jerith666**
###### Things done

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/misc/gammu/default.nix [hunk 1]
```diff
@@ -8,14 +8,16 @@ with stdenv.lib;
...
+    owner = "gammu";
+    repo = "gammu";
+    rev = "49b0b19f59c740817a9582303b8814d8488dc725";
+    sha256 = "0nxb4k1lfsmfiricjfnyx5wdmw84sj3skxrsdkvrbimz2kz0qzs6";
   };
 
-  patches = [ ./bashcomp-dir.patch ];
+  patches = [ ./bashcomp-dir.patch ./systemd.patch ];
 
   buildInputs = [ python pkgconfig cmake bluez libusb1 curl gettext sqlite libiconv ]
```
Comment (ndowens) [inline pkgs/applications/misc/gammu/default.nix]: Pkgconfig cmake goes in nativeBuildInputs

**Comment by jerith666** [comment]
@ndowens done in c33c72b

**Comment by ndowens** [comment]
Thanks

File: pkgs/applications/misc/gammu/default.nix [hunk 2]
```diff
@@ -8,16 +8,20 @@ with stdenv.lib;
...
-  version = "1.33.0";
+  version = "1.37.91";
 
-  src = fetchurl {
-    url = "mirror://sourceforge/project/gammu/gammu/${version}/gammu-${version}.tar.xz";
-    sha256 = "18gplx1v9d70k1q86d5i4n4dfpx367g34pj3zscppx126vwhv112";
+  src = fetchFromGitHub {
+    owner = "gammu";
+    repo = "gammu";
+    rev = "49b0b19f59c740817a9582303b8814d8488dc725";
```
Comment (rasendubi) [inline pkgs/applications/misc/gammu/default.nix]: This would be better (fetch by tag name):
```
rev = version;
```

**Review by rasendubi — COMMENTED** [review]
Also, the latest release is 1.38.2 now. Why not update to that version directly?

File: pkgs/applications/misc/gammu/default.nix (hunk 2 above)
Comment (ndowens) [inline pkgs/applications/misc/gammu/default.nix]: Yup I agree, I tend to use version or v${version} if possible

**Pull request merged**